51E9/MANIFEST

```diff
@@ -0,0 +1,3 @@
+purpose=xyz.openbmc_project.Software.Version.VersionPurpose.PSU
+version=393141313900
```

Comment:
same question as above?

Reply:
The latest FW IBM_RANIER_BOOTLOAD_2300W_1A62C_2E.2F.2D.hex will show version=453246324432